integration/v7/isolated/update_org_quota_command_test.go

```diff
@@ -83,6 +86,7 @@ var _ = Describe("update-org-quota command", func() {
 			Eventually(session).Should(Say(`paid service plans:\s+%s`, "disallowed"))
 			Eventually(session).Should(Say(`app instances:\s+%s`, appInstances))
 			Eventually(session).Should(Say(`route ports:\s+%s`, reservedRoutePorts))
+			//TODO: add an assertion for log quota information
```

Comment:
When are we supposed to add this assertion? Are we waiting for some changes in CAPI?

Reply:
That test, while being part of the `update-org-quota` command test, shells out to the `org-quota` command for checking the results the create. `update-org-quota` does not have it's own table output.

We didn't want to change the `org-quota` command in this PR in order to keep it more logically aligned to the two commands we wanted to change. However, we wanted to leave a reminder to add an assertion there later.

I believe that we should be able to add that assertion in #2299.